native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/ActionsPopupController.m

```diff
@@ -28,7 +28,7 @@
 //action constants
 NSString *const kActionLogout = @"Logout";
 NSString *const kActionSwitchUser = @"Switch user";
-NSString *const kActionDevSupport = @"Dev support";
+NSString *const kActionDbInspector = @"Inspect db";
```

Comment:
Going back to what it was before the dev support work to keep it consistent with Android / hybrid and react native.